### ISSUE TITLE
Low battery notifications

### ISSUE THREAD

**Issue by lion90** [centic9/headset-charge-indicator#9]
While having the tray icon rules, if I'm running something on fullscreen, the battery can run off with no previous alert.

Having a notification when the battery reaches a certain threshold could be useful.

**Comment by lion90** [comment]
At first I was thinking about an audible alert, but then I found [this approach](https://github.com/Roliga/g930-battery-percentage/blob/master/g930-battery-notify) to show OS-level notifications, like other peripherals might do.

I developed a version of my own following Roliga's path. If you don't find it rude, I could make it more user-friendly (disabled as default + args-enabling it, adding it to the README) and PR it.

**Comment by centic9** [comment]
Yes, such a PR would be highly welcome! We can allow the limit to be set when the alert is triggered.
